### DISCUSSION TITLE
How do I configure the mysql in Herd Pro.

### DISCUSSION THREAD

**Discussion by bobgann** [beyondcode/herd-community#610] [Q&A]
I need to manage things like sql_modes in the MySQL server provided in Herd Pro.   I can change modes for the session using the console, but not persistently.   Normally, I would change them in my.cnf - but I cannot find that file - or a configuration file in Herd Pro.

**Comment by johnluxor** [comment]
You can create my.ini file in mysql directory

C:\Users\[USERNAME]\.config\herd\bin\services\mysql\8.0.36 and set necessary options

For MacOS should be the same, just different path

**Comment by bobgann** [comment]
Thanks @johnluxor 

This worked.  
 For reference, I'm on a Mac, so it is slightly different.

Mac path and file   
/Users/Shared/Herd/services/mysql/8.0.36/my.cnf
Note - my.cnf, not my.ini

Windows
C:\Users[USERNAME].config\herd\bin\services\mysql\8.0.36 
my.ini



Bob
